CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0.0] 2021-12-14
+### Added
+- `PrivateKey` and `PublicKey` types, offering the following functions:
+```go
+GenerateKey(rnd io.Reader) (sk *PrivateKey, err error) // to generate a new keypair
+(sk *PrivateKey) Public() PublicKey // To export a public key from a private
+(sk *PrivateKey) Bytes() // To serialise a private key
+(sk *PrivateKey) Prove(alpha []byte) (beta, proof []byte, err error) // To generate proof and VRF
```

Comment:
message and vrf are the commonly used ones I've seen, is that fine?

Reply:
Sure :+1: (I mean, personally I think `vrf` is potentially confusing because the entire function is the VRF, so it's a bit weird to name only one of its outputs `vrf`, but OK if it's common usage then it's probably fine)